addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -350,6 +350,25 @@ $o-wslides-fs-side-width: 300px;
             }
         }
 
+        .o_wslides_slide_pulse {
+            animation: pulse .6s linear normal;
+        }
+
+        @keyframes pulse {
```

Comment:
I would name it slidePulse or boxShadowPulse to be more specific. pulse seems a bit too generic. #longtermthinking